content.yml

```diff
@@ -60,15 +60,165 @@ team:
   visible: true
   title: "Team"
   intro: "Team of entrepereneurs and experienced professionals."
+  block: "../blocks/person"
   members:
-    block: "../blocks/person"
-    name: "Mostafa Balata"
-    title: "Software developer"
-    image: "../assets/mostafabalatabw.jpg"
-    preTitle: "Partner"
-    bio:  |
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+   - name: "Zoe Adamovicz"
+     title: "CEO and Co-Founder"
+     image: "../assets/ZoeAdamovicz_bw.jpg"
+     preTitle: ""
+     domain: ""
+     email: "writeme@neufund.org"
+     bio:  |
+       "Zoe Adamovicz is the CEO and co-founder at Neufund. She is an experienced entrepreneur and occasional angel investor.
```

Comment:
Yes, they are something like this
![image](https://user-images.githubusercontent.com/23242822/29622881-e51fc478-8824-11e7-8fdc-0e9072f08f47.png)